src/api/integrations/chatbot/evoai/validate/evoai.schema.ts

```
@@ -0,0 +1,115 @@
+      (properties[property] = {
+        minLength: 1,
+        description: `The "${property}" cannot be empty`,
+      }),
+  );
+  return {
+    if: {
+      propertyNames: {
+        enum: [...propertyNames],
+      },
+    },
+    then: { properties },
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)